lib/hanami/helpers/form_helper/form_builder.rb

```diff
@@ -1284,10 +1292,11 @@ def select(name, values, attributes = {}) # rubocop:disable Metrics/AbcSize, Met
           attributes  = { name: _select_input_name(name, multiple), id: _input_id(name) }.merge(attributes)
           prompt      = options.delete(:prompt)
           selected    = options.delete(:selected)
+          disabled    = prompt.nil? ? nil : DISABLED
           input_value = _value(name)
 
           super(attributes) do
-            option(prompt) unless prompt.nil?
+            option(prompt, disabled: disabled) unless prompt.nil?
```

Comment:
I think we can remove all the other changes to this file and just do this:
```suggestion
            option(prompt, disabled: true) unless prompt.nil?
```

The `lib/hanami/helpers/html_helper/empty_html_node.rb` file takes care of changing `disabled: true` to be the html `disabled="disabled"` so we don't need to duplicate that logic here.

(Also I know it was already here, but I personally like `if prompt` better, since if prompt is `false`, it's not `nil`, but I think we'd want the same behavior, since both are false-y)

Reply:
Yeah good shout, I'll make the changes

Reply:
Ok I made the requested changes and squashed the commits into one, 5 seemed a little excessive for 6 lines of code :)